pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -525,3 +529,53 @@ def test_all(self):
             RPM_UNSIGNED_FEED_COUNT,
             repo['content_unit_counts']['rpm'],
         )
+
+
+class SyncZchunkRepoSkipTestCase(unittest.TestCase):
+    """Sync feed with ``zchunks`` and ensure no ``zchunk`` units exist.
+
+    A new compression type for repodata exists in Fedora 30 called
+    ``zchunks``. These can be created with patches made to ``createrepo_c --zck``.
+
+    Pulp 2 will ignore this repodata archive type to preserve the quality of the
+    existing data. At the time of writing this test, ``zchunk`` data is
+    in addition to standard repodata. There are currently no cases of
+    repodata only containing ``zchunk`` archives.
+
+    This test targets the following issues:
+
+    `Pulp #4530 <https://pulp.plan.io/issues/4529>`_
+    `Pulp #4530 <https://pulp.plan.io/issues/4530>`_
+
+    Steps:
+
+    1. Clean all existing orphans to prevent false failures.
+    2. Create a repo point to a ``zchunk`` feed and sync it.
+    3. Assert no local units with the ``.zck`` extension.
+
+    """
+
+    def test_zchunk_sync(self):
+        """Sync a ``zchunk`` repo and verify Pulp 2 does no sync ``.zck`` data."""
+        cfg = config.get_config()
+        if check_issue_4529(cfg):
+            self.skipTest('https://pulp.plan.io/issues/4529')
+        client = api.Client(cfg, api.json_handler)
+        client.delete(ORPHANS_PATH)
+
+        # Sync Repo
+        body = gen_repo(
+            importer_config={'feed': ZCHUNK_FEED_URL},
+            distributors=[gen_distributor(auto_publish=True)]
```

Comment:
`auto_publish=True` will force a publish to happen. We should either update docstrings, or remove this.